stories/components/form/ArrayList.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta, Canvas, Story, ArgsTable, Source } from '@storybook/addon-docs';
+import ArrayList from '@/pkg/rancher-components/src/components/Form/ArrayList/ArrayList';
+import { _EDIT, _VIEW } from "@shell/config/query-params";
+
+export const Template = (args, { argTypes }) => ({
+  components: { ArrayList },
+  props:      Object.keys(argTypes),
+  template:   '<ArrayList v-bind="$props" />',
```

Comment:
We should also bind the props for events as follow, so can be observed the emission of the value.
[From the docs](https://storybook.js.org/docs/vue/writing-stories/args#story-args).

```
  template:   '<ArrayList v-bind="$props" v-on="$props"/>',
```

It will require to list them  on  the `Meta` in something like:

```
  argTypes={{ 
    add: {
      action: 'add',
    },
  }}
```

Reply:
I don't see what's the point of listing emitted events here. What's the use case? Who would use Storybook to find out events? For me, I would always just look at the actual Vue component 100 percent of the time.

Reply:
Seems like there's a way to automatically add that, so we can skip this point.
We can discuss about the style guide use in another time and agree on some guideline as a team, since I don't think is the place and is not something I can choose on my own.
Sorry if it bothered you.